resources/liquibase/db.changelog.xml

```diff
@@ -125,6 +126,49 @@
             schemaName="public"
             tableName="books"/>
     </changeSet>
+
+    <changeSet id="202309030849" author="moroti">
+      <createTable tableName="borrowings">
+          <column name="id" type="int" autoIncrement='true'>
+            <constraints primaryKey="true"/>
+          </column>
+          <column name="user_id" type="int">
+            <constraints references="users(id)" foreignKeyName="fk_borrowings_user_id"/>
+          </column>
+          <column name="book_id" type="int">
+            <constraints references="books(id)" foreignKeyName="fk_borrowings_book_id"/>
+          </column>
+           <column defaultValueBoolean="false" name="isReturned" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+           <column name="returned_date" type="datetime" defaultValue="NULL">
+                <constraints nullable="true" />  
```

Comment:
It should not be nullable. 

In addition, if you want to make a filed, nullable, no need specify the nullable constraint cuz not nullable is the default behaviour.